packages/superset-ui-chart/package.json

```diff
@@ -27,6 +27,8 @@
     "access": "public"
   },
   "dependencies": {
+    "@superset-ui/translation": "^0.12.21",
```

Comment:
These should be `peerDependencies`. (`translation` needs to be a singleton. `validator` not necessariliy, but I have been making all `@superset-ui/xxx` `peerDependencies` by convention to avoid having multiple copies of some singleton by mistake)